tests/steps/speculative_execution_test.go

```diff
@@ -60,10 +60,6 @@ func InitializeSpeculativeExecution(ctx *godog.ScenarioContext) {
 		})
 
 	ctx.Step(`^a valid speculative_exec_result will be returned with (\d+) transforms$`, func(transformCount int) error {
-		if len(speculativeExecResult.DeployHash.String()) == 0 {
```

Comment:
This should be changed to test correct data structure BlockHash

Reply:
done